Packs/Legacy/Scripts/PcapHTTPExtractor/PcapHTTPExtractor.py

```diff
@@ -195,6 +195,18 @@ def fix_order(http_packets):
     return new_order
 
 
+def process_tcp_stream(tcp_packets):
```

Comment:
Im not familiar with some of the functions used here, any reason that they will fail? 
If so, maybe add some error handling mechanism here?